plugins/modules/net_tools/gandi_livedns.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019 Gregory Thiemonge <gregory.thiemonge@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: gandi_livedns
+author:
+- Gregory Thiemonge (@gthiemonge)
+requirements:
+- python >= 2.6
+version_added: "1.4.0"
+short_description: Manage Gandi LiveDNS records
+description:
+- "Manages dns records via the Gandi LiveDNS API, see the docs: U(https://doc.livedns.gandi.net/)"
```

Comment:
```suggestion
- "Manages dns records by the Gandi LiveDNS API, see the docs: U(https://doc.livedns.gandi.net/)"
```
or `through` or anything else except latin phrases, they shouldn't be used according to Ansible doc guide